charters/DocumentationCharter.md

```diff
@@ -0,0 +1,54 @@
+# Docs Working Group Charter (Proposed)
```

Comment:
In the Jupyter ecosystem, the term "doc" has a number of different meanings (notebook docs, documentation, etc.). It may be helpful to spell out "Documentation" in the charter to clarify this to people who may not have full context.